aicsimageio/readers/reader.py

```diff
@@ -114,11 +115,36 @@ def is_this_type(cls, data: types.ImageLike) -> bool:
     def _is_this_type(buffer: io.BytesIO) -> bool:
         pass
 
-    @property
     @abstractmethod
-    def dask_data(self) -> da.core.Array:
+    def _build_delayed_dask_data(self) -> da.core.Array:
+        pass
+
+    @abstractmethod
+    def _read_in_memory_data(self) -> np.ndarray:
         pass
 
+    @property
+    def dask_data(self) -> da.core.Array:
+        if self._dask_data is None:
+            try:
+                # These lines both check if distributed has been imported
+                # and if a client connection has been created
+                # If distributed hasn't been imported it will KeyError
+                # If no client has been created it will ValueError
+                # We can't import distributed due to it requiring network utilities:
+                # https://github.com/AllenCellModeling/aicsimageio/issues/82
+                sys.modules["distributed"].get_client()
```

Comment:
didn't we look at this together and decide that this check was not sufficient?

Reply:
We did. But then I read through the distributed codebase and the functions that we talked about at some point route to `get_client` anyway which in turn routes to `_get_default_client` but since that latter is an underscore method I felt better about using `get_client` because it may be more stable release to release.

Reply:
As long as it works!